libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -1517,26 +1517,26 @@ void GCS_MAVLINK::send_home(const Location &home) const
     }
 }
 
-void GCS_MAVLINK::send_home_all(const Location &home)
+#define FOR_EACH_ACTIVE_CHANNEL(methodcall)          \
+    do {                                             \
+        for (uint8_t i=0; i<num_gcs(); i++) {        \
+            if (!chan(i).initialised) {              \
+                continue;                            \
+            }                                        \
+            if (!(GCS_MAVLINK::active_channel_mask() & (chan(i).get_chan()-MAVLINK_COMM_0))) { \
+                continue;                            \
+            }                                        \
+            chan(i).methodcall;                      \
+        }                                            \
+    } while (0);
+
+void GCS::send_home(const Location &home) const
```

Comment:
Given we already have a GCS.cpp file should this go there? I think it would be better, easier to find.